code/src/reducers/directions.js

```diff
@@ -0,0 +1,67 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+import { API_START, API_MOVE } from '../reusables/urls'
+
+export const directions = createSlice({
+  name: "directions",
+  initialState: {
+    name: '',
+    player: '',
+    step: localStorage.getItem('directions') 
+      ? JSON.parse(localStorage.getItem('directions'))
```

Comment:
nice that you saved game status, but You need to remember to add an easy way to remove it. For example, after finishing the game or on click event in restart button. Your users are not IT people and they will not know how to clean local storage from console level.